### PR TITLE
update kubectl apply help info

### PR DESCRIPTION
**Pull request by adohe-zz**
Please refer #22342 for more detail. @bgrant0607 ptal. Also I have open a PR to update docs on `kuberntes.github.io`

### CODE REVIEW

**Comment by bgrant0607** [comment]
Thanks much. LGTM.
cc @janetkuo 

Not sure how we want to update kubernetes.github.io, in general. cc @johndmulhausen

**Comment by bgrant0607** [comment]
Please run hack/update-generated-docs.sh

File: docs/user-guide/kubectl/kubectl_apply.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Apply a configuration to a resource by filename or stdin
 
 Apply a configuration to a resource by filename or stdin.
 The resource will be created if it doesn't exist yet.
+If you want to use apply, please always use apply to create the resource initially or use create --save-config.
```
Comment (janetkuo) [inline docs/user-guide/kubectl/kubectl_apply.md]: Suggest rephrase it to -- `To use 'apply', always create the resource initially with either 'apply' or 'create --save-config'.`

**Comment by janetkuo** [comment]
@AdoHe What's the PR number for docs update?

**Comment by AdoHe** [comment]
@janetkuo just update this. The PR number of doc update is [463](https://github.com/kubernetes/kubernetes.github.io/pull/463)

**Comment by janetkuo** [comment]
@AdoHe thanks!

**Comment by AdoHe** [comment]
@janetkuo please re-add LGTM label.
